app/gateway/2.7.x/admin-api/licenses/reference.md

```diff
@@ -161,41 +184,95 @@ HTTP 200 OK
 }
 ```
 
-### List a License
+## Delete a license
 
 **Endpoint**
 
-<div class="endpoint get">/licenses/{id}</div>
+<div class="endpoint delete">/licenses/{id}</div>
 
 {{ page.licenses_attribute_id }}
 
 **Response**
 
 ```
-HTTP 200 OK
+HTTP 204 No Content
 ```
 
-```json
-{
-  "created_at": 1500508800,
-  "id": "30b4edb7-0847-4f65-af90-efbed8b0161f",
-  "payload": "{\"license\":{\"payload\":{\"admin_seats\":\"1\",\"customer\":\"Example Company, Inc\",\"dataplanes\":\"1\",\"license_creation_date\":\"2017-07-20\",\"license_expiration_date\":\"2017-07-21\",\"license_key\":\"00141000017ODj3AAG_a1V41000004wT0OEAU\",\"product_subscription\":\"Konnect Enterprise\",\"support_plan\":\"None\"},\"signature\":\"24cc21223633044c15c300be19cacc26ccc5aca0dd9a12df8a7324a1970fe304bc07b8dcd7fb08d7b92e04169313377ae3b550ead653b951bc44cd2eb59f6beb\",\"version\":\"1\"}}",
-  "updated_at": 1500508800
-}
-```
+## Generate a report
 
-### Delete a License
+Generate a report on the Kong Gateway instance to gather monthly usage data.
 
-**Endpoint**
+<div class="endpoint get">/licenses/report</div>
```

Comment:
```suggestion
<div class="endpoint get">/license/report</div>
```

Reply:
I think it is not super-odd to not have this documented before, this endpoint was used only when Kong representatives asked customers to send their usage report if I understood correctly. But as this is available we should document it and I agree with you, let's keep it with the other license manipulation docs (with this small fix).